README.md

```diff
@@ -106,6 +106,25 @@ Use your favorite method to build it on your system, for example by using
 `aurutils`:
 ```bash
 aur sync kubeone && pacman -S kubeone
+#### Docker
+
+We distribute a docker image that contains KubeOne and Terraform to use for your
+setup.
+
+##### Building
+
+```bash
+docker build -t kubeone:<Tag> .
+```
+
+##### Running
+
+To receive SSH credentials KubeOne needs to connect to the local SSH
+agent. This is done by mounting its UNIX socket. You can execute KubeOne and
+Terraform that way by running the following command line:
+
+```bash
+docker run --rm --user $(id -u) --name kubeone --mount type=bind,source="$SSH_AUTH_SOCK",target=/run/ssh.sock --mount type=bind,source="$(pwd)",target=/mnt -t kubeone:<Version> <Command>
```

Comment:
oh well.. all the ENV with credentials is missing, and I suppose before we get "secrets in file" it will be very cumbersome to list every single ENV in CLI

Reply:
This is my problem as well .... kubeone in a container seems rather useless since you need to work around the sandboxing.